dotnet/Controllers/RoutesController.cs

```diff
@@ -182,6 +182,18 @@ public async Task<IActionResult> ProcessReviewApiAction(string requestedAction,
                 var orderBy = queryString["order_by"];
                 var status = queryString["status"];
                 var productId = queryString["product_id"];
+                var locale = queryString["locale"];
+                var boolValue = queryString["pastReviews"];
```

Comment:
This could be replaced with
var boolValue = queryString["pastReviews"];
bool pastReviews = false;
bool.TryParse(boolValue, out pastReviews);